cmd/config/internal/commands/run_test.go

```diff
@@ -183,7 +183,7 @@ apiVersion: v1
 				"--star-name", "foo",
 				"--", "Foo", "g=h"},
 			path: "dir",
-			err:  "must specify --star-path with --enable-star",
+			err:  "must specify --enable-star with --star-path",
```

Comment:
I'm curious why `--enable-star` must be specified.

Reply:
Can't we imply it as enabled if star-path is specified.

Reply:
Because the feature is alpha, and disabled by default.